Sources/Vortex/Helpers/Angle+Codable.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Angle+Codable.swift
+//  Vortex
+// https://www.github.com/twostraws/Vortex
+// See LICENSE for license information.
+//
+
+import SwiftUI
+
+/// Store the radians value of Angle for decoding/encoding.
+extension Angle: Codable {
```

Comment:
Apple is rightly not keen on people extending types they don't own to protocols they don't own, so from Swift 6 onwards we need to mark this as a [retroactive conformance](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0364-retroactive-conformance-warning.md). The actual solution here is for Apple to make `Angle` conform to `Codable` for us, along with other common, trivial types such as `UnitPoint` and `CLLocationCoordinate2D`, because they are pain points right now.

However, in the meantime I think we should proceed with your suggestion, as long as you add the `@retroactive` marker.